inc/saitypes.h

```diff
@@ -1196,6 +1199,54 @@ typedef struct _sai_port_snr_list_t
     sai_port_snr_values_t *list;
 } sai_port_snr_list_t;
 
+/**
+ * @brief poe port active channel (when delivering power)
+ */
+typedef enum _sai_poe_port_active_channel_id_t
+{
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_A = 0,
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_B = 1,
+    SAI_POE_PORT_ACTIVE_CHANNEL_TYPE_A_AND_B = 2
+} sai_poe_port_active_channel_id_t;
+
+/**
+ * @brief poe port signature type (when delivering power) 
+ */
+typedef enum _sai_poe_port_signature_type_id_t
+{
+    SAI_POE_PORT_SIGNATURE_TYPE_SINGLE = 0,
+    SAI_POE_PORT_SIGNATURE_TYPE_DUAL = 1
+} sai_poe_port_signature_type_id_t;
+
+/**
+ * @brief poe port classification method (when delivering power)
+ */
+typedef enum _sai_poe_port_class_method_id_t
+{
+    SAI_POE_PORT_CLASS_METHOD_TYPE_REGULAR = 0,
+    SAI_POE_PORT_CLASS_METHOD_TYPE_AUTO_CLASS = 1
+} sai_poe_port_class_method_id_t;
+
+/**
+ * @brief Defines a port consumption structure
+ * 
+ * Data that is needed and available when a port is delivering power
+ */
+typedef struct _sai_poe_port_power_consumption_t {
+
+    /** Port power consumption value */
+    sai_poe_port_active_channel_id_t active_channel; /* active channel a/b/ab */
+    uint32_t voltage_in_volts; /* volts */
```

Comment:
Unit should not be contained in field but in description, also volts resolution Is probably too big you probably need mV

Reply:
Yes, they should all be milli. Thanks!